jj4x4: README update (#7854)
qmk/qmk_firmware: keyboards/jj4x4/README.md

    make jj4x4:default:flash

**Reset Key**:

> Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard USB cable.

Reviewer: ```suggestion
Hold down the key located at *K12*, commonly programmed as *8* while plugging in the keyboard USB cable.
```

My JJ4x4 arrived, it is indeed this key, but it is labelled K12 in the macro define.

> Key *K00* is in the above picture:

Reviewer: ```suggestion
Key *K12* is in the above picture:
```

Row 2 from the top, Column 2 from the left, see the schematic below:

```
  ,-----------------------.
  |     |     |     |     |
  |-----`-----`-----`-----|

>   |     | K00 |     |     |

Reviewer: ```suggestion
  |     | K12 |     |     |
```

> Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard usb cable.

Reviewer 2: ```suggestion
Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard USB cable.
```

> Row 3 from bottom (or 1 from top), Column 2 from left (or 3 from right), just like the schematic below:

Reviewer 2: ```suggestion
Row 2 from the top, Column 2 from the left, see the schematic below:
```
This is optional but makes more sense in my opinion

Author: sure, it's more simple.
